client/src/components/application/ResponsiveDrawer/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Drawer from '@material-ui/core/Drawer';
+import Hidden from '@material-ui/core/Hidden';
+import PropTypes from 'prop-types';
+import DrawerList from './list';
+import useStyles from './style';
+
+const ResponsiveDrawer = ({ Content, IsFinished }) => {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <Hidden implementation="js">
+        <Drawer
+          classes={{
+            paper: classes.drawerPaper,
+          }}
+          variant="persistent"
+          open
+        >
+          <DrawerList content={Content} isFinished={IsFinished} />
+        </Drawer>
+      </Hidden>
+    </div>
+  );
+};
+
+ResponsiveDrawer.propTypes = {
+  Content: PropTypes.node,
+  IsFinished: PropTypes.node,
```

Comment:
may be the type here is array of string.

Reply:
yes, the type is an array, and as written in react docs, you can use the node to render array or string or element.
and here I want to use elements inside the array 
[react docs](https://reactjs.org/docs/typechecking-with-proptypes.html).